### ISSUE TITLE
Screen rotation broken on Onyx Poke 5

### ISSUE THREAD

**Issue by ilyats** [koreader/koreader#10983]
* KOReader version: 2023.08-50
* Device: Onyx Poke 5

#### Issue
Changing screen orientation doesn't work
#### Steps to reproduce
Open PDF file ,  get  to menu, select landscape orientation  or upside down portrait orientation from the menu at  the bottom.
The orientation of the text does not change, but behaviour of  Koreader changes:  screen fit settings (Full/width/height) and "zoom to" settings get disabled, and Koreader behaves as "Fit" setting were set to "width"

I am attaching output of adb logcat with verbose debugging log enabled in Koreader
[rotate.log](https://github.com/koreader/koreader/files/12831426/rotate.log)

**Comment by hugleo** [comment]
duplicated

Track progress here: https://github.com/koreader/android-luajit-launcher/pull/439

**Comment by ilyats** [comment]
@hugleo is this the  same bug?  My device has Android 11 (or at least claims so)

**Comment by NiLuJe** [comment]
But it's an Onyx, so all bets are off ;).

**Comment by hugleo** [comment]
> @hugleo is this the same bug? My device has Android 11 (or at least claims so)

Yes, it's the same version as mine. Ultrac is also on Android 11. For utlrac IIRC I got rid of this problem using app optimizations.
